test/foundry/unit/StakingRewardsV2/StakingRewardsV2OnBehalfActionsTests.t.sol

```diff
@@ -356,18 +286,20 @@ contract StakingRewardsV2OnBehalfActionsTests is DefaultStakingV2Setup {
     }
 
     /*//////////////////////////////////////////////////////////////
-                    Get Reward And Stake On Behalf
+                    Get Reward On Behalf
     //////////////////////////////////////////////////////////////*/
 
-    function test_Get_Reward_And_Stake_On_Behalf() public {
+    function test_Get_Reward_On_Behalf() public {
```

Comment:
its good that we are testing getReward() by itself now (im assuming we didnt before) but why are we just replacing the "on behalf" test? we no longer test getting the reward AND staking anymore

we should have both this test & the test_Get_Reward_And_Stake. not either or

Reply:
this reminds me: where do we test getReward (at least the new changes)? cant find it